File: go/ex1_3.go
package main

func isPositive(num int) bool { return num > 0 }

func printString(str string) {}

func add(num1 int, num2 int) int { return num1 + num2 }

func main() {
    var res int
    res = 3
    _ = res
    
    isPositive(1)
    printString("2")
    add(1, 2)
}

File: ocaml/ex1_3.ml
let is_positive (num: int): bool = num > 0

let print_string (str: string): unit = ()

let add (num1: int) (num2: int): int = num1 + num2

let _ =
  let res: int = 3 in
  is_positive 1;
  print_string "2";
  add 1 2
